openlibrary/tests/core/test_fulltext.py

```diff
@@ -0,0 +1,33 @@
+import requests
```

Comment:
Nice addition of new tests!!!

Please format new files with psf/black and `isort --profile black`